siocli/lib/SqlDump.cpp

```diff
@@ -42,7 +42,7 @@ struct DatabaseInfo {
     BinaryValue m_cipherKey;
 };
 
-struct ColumnConstaint {
+struct ColumnConstraint {
     ConstraintType m_type;
```

Comment:
All these data structures miss doc comments for each member. REQUIRED by our DG. Please add.